media/css/stories/includes/lib.scss

```diff
@@ -0,0 +1,185 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@use 'sass:math';
+
+// use math.div to get percentages of grid to use for padding and margins
+$column-width: math.percentage(math.div(math.div(100, 12), 100));
+$half-column: math.percentage(math.div(math.div(100, 24), 100));
+$font-stories-title: 'Antonio', sans-serif;
+$font-stories-title-alt: 'League Gothic', sans-serif;
+$font-stories-body: 'Source Serif Pro', sans-serif;
+
+@font-face {
+    font-family: Antonio;
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+    src:
+        url('/media/fonts/fx-lifestyle/antonio-regular-webfont.woff2') format('woff2'),
+        url('/media/fonts/fx-lifestyle/antonio-regular-webfont.woff') format('woff');
+}
+
+@font-face {
+    font-family: Antonio;
+    font-weight: 700;
+    font-style: normal;
+    font-display: swap;
+    src:
+        url('/media/fonts/fx-lifestyle/antonio-bold-webfont.woff2') format('woff2'),
+        url('/media/fonts/fx-lifestyle/antonio-bold-webfont.woff') format('woff');
+}
```

Comment:
Heads up I'm removing these fonts and the page they were associated with in https://github.com/mozilla/bedrock/pull/11951

If you want to keep the fonts for this new page, I'd suggest moving them from `/fonts/fx-lifestyle/` to `/fonts/stories/`

Reply:
Ok, thanks for the notice!